op-guide/migration.md

```diff
@@ -129,7 +129,7 @@ cd mydumper-linux-amd64
 Use the `mydumper` tool to export data from MySQL by typing the following command:
 
 ```bash
-./bin/mydumper -h 127.0.0.1 -P 3306 -u root -t 16 -F 128 -B test -T t1,t2 -o ./var/test
+./bin/mydumper -h 127.0.0.1 -P 3306 -u root -t 16 -F 128 -B test -T t1,t2 --skip-tz-utc -o ./var/test
```

Comment:
@iamxy I think we should update our mydumper source and enable skip-tz-utc in default.